Consul/Tests/Services/KVTest.php

```diff
@@ -82,7 +80,7 @@ public function testDeleteWithDefaultOptions()
             $this->fail('fail because the key does not exist anymore.');
         } catch (\Exception $e) {
             $this->assertInstanceOf('SensioLabs\Consul\Exception\ClientException', $e);
-            $this->assertContains('404 - Not Found', $e->getMessage());
```

Comment:
Symfony has no equivalent for `getReasonPhrase` so I've removed this functionality here in the test and in the client.